OpenSim/Simulation/Model/MovingPathPoint.cpp

```diff
@@ -162,7 +162,7 @@ void MovingPathPoint::updateFromXMLNode(SimTK::Xml::Element& aNode, int versionN
         XMLDocument::renameChildNode(aNode,"ZAttachment", "z_location");
     }
     if (documentVersion < 30505) {
-        // replace old properties with latest use of Connectors
+        // replace old properties with latest use of Sockets
```

Comment:
This should remain as `Connectors` for this model version to first be updated to connectors and then to proceed to the latest update.